ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -26,6 +26,15 @@ build_root:
     namespace: ci
     tag: master
 images:
+- dockerfile_path: images/ci/Dockerfile
+  from: base
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      paths: null
+  optional: true
+  to: launch-rhel-fips
```

Comment:
What I do not know is how will the periodic job I've added below have access to this `${LOCAL_IMAGE_LAUNCH_RHEL_FIPS}`?  @smarterclayton ?

Reply:
to run this job, we need this image: https://github.com/openshift/origin/blob/2ec8c7d7d952a4099e09d18d1da5e0ef97fbf067/images/ci/Dockerfile  to be available - and w/ this pr it will be built with ci-op/config/openshift/origin/openshift-origin..yaml